### PR TITLE
Handle negative numbers in aci

### PR DESCRIPTION
**Pull request by UlfNorell**
Fixes aeternity/aesophia_http#59

### CODE REVIEW

**Comment by tolbrino** [comment]
Changelog entry missing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - FATE code generator improvements.
 - Bug fix: Handle qualified constructors in patterns.
 - Bug fix: Allow switching also on negative numbers.
+- Bug fix: Fixed ACI encoder to handle `-` unary operator
```
Comment (tolbrino) [inline CHANGELOG.md]: Wrong place, as this is unreleased.

Comment (tolbrino) [reply CHANGELOG.md]: Also, seems like the changelog is out of date, should be fixed as part of the next point release.

Comment (radrow) [reply CHANGELOG.md]: I will update the other changes in changelog as a separate PR

Comment (hanssv) [reply CHANGELOG.md]: This is intentional... We update the changelog on every release, not on each commit...

Comment (tolbrino) [reply CHANGELOG.md]: I mixed up different repos. The changelog isn't out of date, my bad. But still the location of the entry was incorrect.

**Pull request merged**